yt/frontends/gadget/tests/test_outputs.py

```diff
@@ -57,6 +58,18 @@ def test_GadgetDataset():
     assert isinstance(data_dir_load(LE_SnapFormat2), GadgetDataset)
 
 
+@requires_file(keplerian_ring)
+def test_NonCosmoDataset():
```

Comment:
nit: PEP8 says this should be `test_non_cosmo_dataset` since this is a function.

Reply:
I completely agree with your nit! I was going for consistency with the above test but I've now refactored them both to use underscores rather than camel.